internal/performance/api_load_test.go

```diff
@@ -0,0 +1,201 @@
+package performance_test
+
+import (
+	"fmt"
+	"testing"
+	"time"
+
+	"github.com/smartcontractkit/chainlink/internal/performance"
+	"github.com/smartcontractkit/chainlink/internal/cltest"
+	"github.com/stretchr/testify/assert"
+	"github.com/tsenart/vegeta/lib"
+)
+
+type Requirement struct {
+	requestsPerSecond int
+	averageLatencyMax time.Duration
+	percentileMax     time.Duration
+	runDurationMax    time.Duration
+}
+
+var (
+	r            Requirement
+	requirements = []Requirement{
+		{20, time.Millisecond * 100, time.Millisecond * 100, time.Millisecond * 100},
+		{40, time.Millisecond * 200, time.Millisecond * 200, time.Millisecond * 200},
+		{60, time.Millisecond * 300, time.Millisecond * 300, time.Millisecond * 300},
+	}
+
+	app     *cltest.TestApplication
+	cleanup func()
+)
+
+func Setup() func(b *testing.B) {
+	 return func(b *testing.B) {
+		for _, req := range requirements {
+			app, cleanup = cltest.NewApplicationWithKeyStore()
```

Comment:
Looks like `cleanup` doesn't need to be global, as it's only used in this function.